src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.ts

```diff
@@ -424,8 +424,9 @@ export class OnlineGameWrapperComponent extends GameWrapper<MinimalUser> impleme
             const currentPartTurn: number = this.gameComponent.getTurn();
             const chosenMove: Move = this.gameComponent.encoder.decode(listMoves[currentPartTurn]);
             const legality: MGPFallible<unknown> = rules.isLegal(chosenMove, this.gameComponent.getState());
+            const stringListMoves: string = '[' + JSON.stringify(listMoves) + ']';
```

Comment:
ça fait pas deux [ et deux ] du coup ?

Reply:
keuwa ?
ça faisait un superbe "in [Object object] [Object object] [Object object]" avant en tout cas, là on VOIT le move, du coup !
Par contre pour notre relisibilité p-e que display that serait mieux (plus manipulable qu'un string simple) ?

Reply:
`JSON.stringify([1, 2, 3])` -> `[1,2,3]`
`'[' + JSON.stringify([1,2,3]) + ']'` -> `[[1,2,3]]`
display keuwa ?

Reply:
bah qu'à un moment, les fonctions (assert par exemple) qui throwent et logguent en db, log en console en plus, içi surtout, logguer la liste (console.log(liste)) valent milles stringifications de liste

Reply:
ah qu'un assert foireux log ça ait du sens oui ! T'peux même le ticketter

Reply:
Non je disais que si ça arrive, un log en console est plus manipulable (et oui le log doit contenir du bon stringifié, pas de l'[Object object] mais c't'une autre question ça.

Reply:
oui un log de l'objet est plus manipulable mais assert prends une string donc c'est inutile c'que tu racontes !

Reply:
non un  log de l'object s't'un string, tu manipule RIEN, si tu console.log(maListDObjet) PAF, là t'as de la manipulabilité